assets/js/donut.js

```diff
@@ -61,20 +59,14 @@ label.append("tspan")
 	.attr("dy", "1.1em")
 	.text("complete");
 
-// building a legend
-svg.append('g')
-    .attr('class', 'legend')
-    .selectAll('text')
-    .data(slices)
-    .enter()
-    .append('text')
-    .text(function (d) {
-        return d.data.value + ' hours - ' + d.data.label;
-    })
-    .attr('fill', function (d) {
-        return color(d.data.label);
-    })
-    .attr('y', function (d, i) {
-        return 20 * (i + 1);
-    })
-    .attr('x', 300);
+(function ($) {
+	$(document).ready(function () {
+		$.each(data,function(i, val){
+			var li = $('<div/>').attr('class','legend-item').appendTo('.donut-legend');
+			$('<span/>').attr('class','square').css('background-color',color(val.label)).appendTo(li);
+			var percentHours = parseFloat(val.value.replace(/,/g, ''))/totalHours*100;
+			var label = $('<p/>').appendTo(li).text(percentHours.toFixed(1) + '% - '+val.label + ' (' + val.value +' hours)');;
```

Comment:
- please remove extraneous semi-colon
- `var label = ` is an unused local variable label and can be removed
- empty line `69` can be removed